samples/bluetooth/fast_pair/input_device/sample.yaml

```
@@ -1,8 +1,8 @@
 sample:
-  name: Fast Pair Provider
-  description: Bluetooth Fast Pair Provider sample
+  name: Fast Pair Provider input device
```

I would use the same name as is defined in the sample Readme file.

```suggestion
  name: Fast Pair Input Device
```